packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -431,9 +429,7 @@ class _DatePickerModeToggleButtonState extends State<_DatePickerModeToggleButton
 
   @override
   Widget build(BuildContext context) {
-    final ColorScheme colorScheme = Theme.of(context).colorScheme;
-    final TextTheme textTheme = Theme.of(context).textTheme;
-    final Color controlColor = colorScheme.onSurface.withOpacity(0.60);
+    final TextStyle? subHeaderTextStyle = DatePickerTheme.of(context).subheaderTextStyle;
```

Comment:
This doesn't look correct? This seems never default to `colorScheme.onSurface.withOpacity(0.60);`, right?

I think it should be something like:
```dart
final DatePickerThemeData datePickerTheme = DatePickerTheme.of(context);
final DatePickerThemeData defaults = DatePickerTheme.defaults(context);
final TextStyle? effectiveSubheaderTextStyle = datePickerTheme.subheaderTextStyle ?? defaults.subheaderTextStyle;
```